src/Javis.jl

```diff
@@ -1206,6 +1214,11 @@ function javis(
         CURRENT_ACTION[1] = actions[1]
     end
 
+    if liveview == true
+        _javis_viewer(video, length(frames), actions)
+        return "Live preview of $pathname"
```

Comment:
I think the `return` feels a bit wrong as `$pathname` doesn't exist anymore.

Reply:
Yea that makes sense. I just made the line simpler.